docs/dev/table/sourceSinks.md

```diff
@@ -714,10 +714,11 @@ connector.debug=true
 
 For a type-safe, programmatic approach with explanatory Scaladoc/Javadoc, the Table & SQL API offers descriptors in `org.apache.flink.table.descriptors` that translate into string-based properties. See the [built-in descriptors](connect.html) for sources, sinks, and formats as a reference.
 
-A connector for `MySystem` in our example can extend `ConnectorDescriptor` as shown below:
-
 <div class="codetabs" markdown="1">
 <div data-lang="java" markdown="1">
+
+A connector for `MySystem` in our example can extend `ConnectorDescriptor` as shown below:
```

Comment:
This isn't exactly true, the class MySystem doesn't extend ConnectorDescriptor. What about: 

> A custom descriptor can be defined by implementing the `ConnectorDescriptor` interface.

Reply:
I realize this isn't strictly part of this ticket but since you're moving this sentence around its a good chance to clean up another sentence in the docs.

Reply:
The `ConnectorDescriptor` is an abstract class. Maybe we should change the descriptions to : A custom descriptor can be defined by extending the `ConnectorDescriptor` class?

Reply:
You’re right.